apix-integrationtests/build.gradle

```diff
@@ -13,7 +13,7 @@ subprojects {
     sourceSets {
         main {
             java {
-                srcDir '../../apix-impl/src/integration-test'
+                srcDir '../src/integration-test/java'
```

Comment:
It is a bit strange to put code from the `main` sourceSet here inside the `integration-test` folder? Can't we rename it to `main` and just drop the sourceset configuration then?

Messing with `runtimeClasspath` may still be necessary unfortunately.